### PR TITLE
DAOS-5441 - weekly-testing: IorInterceptDfuseMix: Failing with NO_SPACE

### PR DESCRIPTION
**Pull request by cibervicho**
Test failing with the error "Reserve 1048576 from NVMe failed. 
DER_NOSPACE(-1007): 'No space on storage target'"
In order to get it solved I'm adding one more server node to the
ior_intercept_dfuse_mix.yaml file so there is enough space required
by the test.

The test has to be skipped since because after fixing the NO_SPACE
error a new one was discovered. Ticket DAOS-5857 is tracking this.

### CODE REVIEW

File: src/tests/ftest/io/ior_intercept_dfuse_mix.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def test_ior_intercept_dfuse_mix(self):
         min_mib = int(IorMetrics.Min_MiB)
         mean_mib = int(IorMetrics.Mean_MiB)
 
-        write_x = self.params.get("write_x", "/run/ior/iorflags/ssf/*", 1)
+        write_x = self.params.get("write_x", "/run/ior/iorflags/ssf/*",1)
```
[the remark this answers is not in the document]
Comment (cibervicho) [reply src/tests/ftest/io/ior_intercept_dfuse_mix.py:75]: This one was intentionally introduced to re-push to CI. Fixing in a new commit.

**Comment by cibervicho** [comment]
The test finished with no NO_SPACE error but it failed when comparing results of read operations.

https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3864/2/artifact/Functional/job-2020-11-18T07.52-ebb86ba-io-ior_intercept_dfuse_mix/job.log

The purpose of this test is to run two jobs in parallel: 3 clients without IL and 1 client without IL. Then preloads the Intercept Library and runs 2 jobs again: 3 clients with IL and 1 client without IL.
Finally do comparisons between runs with and without IL and 1 last comparison between both runs in 1 client without IL, and here is where the test fails.

It seems to be an improvement in read operation in the second run than the first one (both without IL) and should be fairly the same performance results:

# Verifying read performance
with_intercept[2][1][max_mib]  = 1478.20  <  without_intercept[2][1][max_mib]  = 753.74 * 1.5 = 1130.61 -> FALSE
with_intercept[2][1][min_mib]  = 1478.20  <  without_intercept[2][1][min_mib]  = 753.74 * 1.5 = 1130.61 -> FALSE
with_intercept[2][1][mean_mib] = 1478.20  <  without_intercept[2][1][mean_mib] = 753.74 * 1.5 = 1130.61 -> FALSE
This is why the test is failing.

At first we thought it was related to the ticket: DAOS-5857, but this case seems to be a little bit different because the performance comparison is between 2 read ops without Intercept Library.

**Review by shimizukko — DISMISSED** [review]
LGTM

**Comment by saurabhtandan** [comment]
The large tests never ran.May be the PR needs to be re-triggered for test stage.

**Comment by cibervicho** [comment]
> The large tests never ran. May be the PR needs to be re-triggered for test stage.

I re-triggered the test stage but that didn't help, the large hw tests didn't run.
I just re-merged with master and pushed again.

**Comment by cibervicho** [comment]
Created ticket DAOS-7068 to fix the 'False is not true' error. I'm skipping this test for now.

**Comment by cibervicho** [comment]
The ticket DAOS-7068 was resolved as a duplicate of DAOS-5857.
I'm pushing another job to CI but skipping the test with the latter one.

**Comment by cibervicho** [comment]
CI checks finished failing Functional_Hardware_Medium only in 'Archive JUnit-formatted test results' step with error:

[2021-03-24T06:10:32.911Z] Recording test results
[2021-03-24T06:10:33.340Z] [Checks API] No suitable checks publisher found.
Functional_Hardware_Medium failed: 1

I checked the results of all tests in this phase in Jenkins and all passed successfully but those under "osa" which were skipped:

Functional_Hardware_Medium / osa / 
 - osa_offline_reintegration.py
 - osa_online_drain.py
 - osa_online_extend.py
 - osa_online_parallel_test.py
 - osa_online_reintegration.py

May I have your code review again please?

**Review by shimizukko — APPROVED** [review]

**Comment by cibervicho** [comment]
Had a chat with Makito Kano and we agreed this ticket could be closed as duplicate. This issue is being tracked in DAOS-6916

**Pull request closed without merging**